### ISSUE TITLE
Runtime error in Json

### ISSUE THREAD

**Issue by giauten** [Masuzu/ZooeyBot#925]
Hi, I'm farming RoTB using scheduling mode with this script:

```
for i=1,3 do
  for i=1,8 do
	  SetLuaScript("Scripts/rotb.lua")
	  SetPreferredSummons("Hades, Celeste Omega_1, Bamamut")
	  RepeatQuest("http://game.granbluefantasy.jp/#quest/supporter/711041/1", 20, 1)
	  Assert4SymbolsPendantsBelow(30000)
  end
  SetPreferredSummons("Godsworn Alexiel_1, Yggdrasil Omega_2")
  SetLuaScript("Scripts/rotb-nep.lua")
  RepeatQuest("http://game.granbluefantasy.jp/#quest/supporter/711051/1/0/5020", 30, 1)
  Assert4SymbolsPendantsBelow(30000)
end
```
everything was working well until I entered primal beast battle, I got this error at the beginning of battle:
![unknown](https://user-images.githubusercontent.com/43314003/45595735-4b2c3e80-b9da-11e8-8392-054c58a3c08c.png)

So I tried change to this one

```
for i=1,3 do 
  SetLuaScript("Scripts/rotb.lua")
  SetPreferredSummons("Hades, Celeste Omega_1, Bamamut")
  RepeatQuest("http://game.granbluefantasy.jp/#quest/supporter/711041/1", 20, 8)

  SetLuaScript("Scripts/rotb-nep.lua")
  SetPreferredSummons("Godsworn Alexiel_1, Yggdrasil Omega_2")
  RepeatQuest("http://game.granbluefantasy.jp/#quest/supporter/711051/1/0/5020", 30, 1)
end
```
but also getting the same error.
the **Scripts/rotb-nep.lua** working fine in Trial Mode so I think it's not the problem

How to fix it pls?

**Comment by giauten** [comment]
Oh it found out it
When I enable Early start with Viramate and have SelectTarget at first command, the bug above will occur

**Comment by Masuzu** [comment]
Hello,
Ah, thank you a lot for reporting this! I will note it down to patch it in the next version.

**Comment by Masuzu** [comment]
This was addressed in v3.0.4
